deps/v8/src/api.cc

```diff
@@ -939,6 +939,15 @@ void Template::Set(v8::Local<Name> name, v8::Local<Data> value,
   i::Isolate* isolate = templ->GetIsolate();
   ENTER_V8(isolate);
   i::HandleScope scope(isolate);
+  auto value_obj = Utils::OpenHandle(*value);
```

Comment:
Oh I see. This is the change which you initially had in #6228, right? [This](https://codereview.chromium.org/1839983002/diff/1/src/api.cc)? Then, we may not need a TODO I think. Eventually v8 upgrades will take care of this.